app/models/location.rb

```diff
@@ -1,4 +1,9 @@
 class Location < G5Updatable::Location
   has_many :phone_numbers
   has_many :ppc_numbers
+
+  def country_code_prefix
+    country_code = properties['country_code'] ? properties['country_code'] : "1"
```

Comment:
I think `country_code = properties['country_code'] || "1"` does pretty much the same thing, doesn't it? Since I think  properties['country_code'] will return nil until the hub starts serving it up